terminal_sunday.sh

```diff
@@ -26,6 +26,10 @@ case "$(uname)" in
     birth_year=$(date -j -f "%Y-%m-%d" "$birthdate" +"%Y")
     birth_timestamp=$(date -j -f "%Y-%m-%d" "$birthdate" +%s)
     ;;
+  "MINGW"*)
```

Comment:
I believe it's possible to place condition above, like: `"Linux"|"MINGW")`

Reply:
Well, true.